src/workers/parser/pausePoints.js

```diff
@@ -98,9 +98,8 @@ function onEnter(node: BabelNode, ancestors: SimplePath[], state) {
 
     if (isCall(value) || t.isFunction(parentNode)) {
       return addEmptyPoint(state, startLocation);
-    } else {
-      return addStopPoint(state, startLocation);
     }
```

Comment:
is this intentional?

Reply:
This may have been a result of `yarn lint`; I didn't do it manually.